pkg/backend/diy/backend.go

```diff
@@ -1350,6 +1354,32 @@ func (b *diyBackend) UpdateStackTags(ctx context.Context,
 	return errors.New("stack tags not supported in diy mode")
 }
 
+func (b *diyBackend) EncryptStackDeploymentSettingsSecret(ctx context.Context,
+	stack backend.Stack, secret string,
+) (string, error) {
+	// The local backend does not support managing deployments.
+	return "", errors.New("stack deployments not supported with diy backends")
+}
+
+func (b *diyBackend) UpdateStackDeploymentSettings(ctx context.Context, stack backend.Stack,
+	deployment apitype.DeploymentSettings,
+) error {
+	// The local backend does not support managing deployments.
+	return errors.New("stack deployments not supported with diy backends")
+}
+
+func (b *diyBackend) DestroyStackDeploymentSettings(ctx context.Context, stack backend.Stack) error {
+	// The local backend does not support managing deployments.
+	return errors.New("stack deployments not supported in --local mode")
+}
+
+func (b *diyBackend) GetStackDeploymentSettings(ctx context.Context,
+	stack backend.Stack,
+) (*apitype.DeploymentSettings, error) {
+	// The local backend does not support managing deployments.
+	return nil, errors.New("stack deployments not supported in --local mode")
```

Comment:
```suggestion
	return nil, errors.New("stack deployments not supported with diy backends")
```

Reply:
I was sure I updated those initially and turns out I missed two 🤦